distribution/docs/src/main/resources/_contents/_catalog-frameworks/default-attribute-values-contents.adoc

```diff
@@ -0,0 +1,69 @@
+
+==== Default Attribute Values
```

Comment:
Not sure if it's correct or what you meant, but I moved those two sections to be closer together in the outline.